type_description_interfaces/msg/FieldType.msg

```diff
@@ -0,0 +1,157 @@
+# Represents the type of a field and related meta-data.
+
+# A constant for each type supported according to:
+#   http://design.ros2.org/articles/legacy_interface_definition.html
+# and:
+#   http://design.ros2.org/articles/idl_interface_definition.html
+# Order is loosely coupled to the order of appearance in the IDL 4.2 spec:
+#  https://www.omg.org/spec/IDL/4.2
+
+# Layout of constants across the 0-255 decimal values in the uint8:
+#
+# - 000    : Reserved for "not set"
+# - 001-048: Primitive types, strings, and reserved space for future primitive types
+# - 049-096: Fixed sized array of primitive and string types
+# - 097-144: Bounded Sequences of primitive and string types
+# - 145-192: Unbounded Sequences of primitive and string types
+# - 193-255: Reserved space for future array/sequence-like types
+
+uint8 FIELD_TYPE_NOT_SET = 0
+
+# Nested type defined in other .msg/.idl files.
+uint8 FIELD_TYPE_NESTED_TYPE = 1
+
+# Basic Types
```

Comment:
```suggestion
```

There are no other separators like this one, so I think it is superfluous

Reply:
Agreed, removed in 2563dc382581359b57d945b3a8b321599e215030